features/topology-map/plugins/org.opennms.features.topology.plugins.topo.graphml/pom.xml

```diff
@@ -103,6 +103,11 @@
       <artifactId>org.osgi.core</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.opennms.features</groupId>
+      <artifactId>org.opennms.features.osgi-jsr223</artifactId>
+      <version>${project.version}</version>
```

Comment:
scope should be provided

Reply:
I updated scope here. But in case of adapters-collection module,  I have it as default scope. We need to have it work on Spring as well if feature doesn't exist, so OSGiScriptEngine related files needs to be in default classpath.